FPIS/Utils/Enums.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace FPIS.Utils
+{
+    // NOTE: This Enums source code file can be used to store all the flags we will use in this codebase
+    
+    /// <summary>
+    /// Custom enum to indicate the different states during login authentication
+    /// </summary>
+    public enum LoginAuth
+    {
+        /// <summary>
+        /// User didn't provide a username
+        /// </summary>
+        NOUSERNAME,
```

Comment:
Can you use a consistent naming convention? For readability 😄  
Suggestion: `NOUSERNAME` => `NO_USERNAME`

Reply:
Lol... my bad. I'll use the one with underscores.

Reply:
I meant the one WITHOUT